opal/mca/common/ucx/common_ucx.h

```diff
@@ -56,6 +50,33 @@ ucs_status_t opal_common_ucx_wait_request(ucs_status_ptr_t request, ucp_worker_h
     }
 }
 
+static inline
+int opal_common_ucx_wait_request_opal_status(ucs_status_ptr_t request, ucp_worker_h worker)
+{
+    /* check for request completed or failed */
+    if (OPAL_LIKELY(UCS_OK == request)) {
+        return OPAL_SUCCESS;
+    } else if (OPAL_UNLIKELY(UCS_PTR_IS_ERR(request))) {
+        return OPAL_ERROR;
+    }
+
+    return opal_common_ucx_wait_request_internal(request, worker) == UCS_OK ?
```

Comment:
this is not helpful, there is still branch in '?:' operator, need to revert or eliminate it by changing opal_common_ucx_wait_request_internal

Reply:
right, but it happened after few progress loops are completed and it could not affect to overal time

Reply:
ok

Reply:
@yosefe actually result of ***_flush is used for diagnostic only... leave opal result only?